From 7e5bcd71d292d8c92cd6390b982b78cee86cb30a Mon Sep 17 00:00:00 2001
From: Santhosh Kumar <santhosh.kumar.selvaraj@qt.io>
Date: Thu, 18 Sep 2025 10:39:15 +0200
Subject: [PATCH 047/239] Mark security header for the qt labs platform

No security critical issues identified for the qt labs platform.

QUIP: 23
Task-number: QTBUG-136184
Pick-to: 6.9 6.8
Change-Id: If2d0e6991463ed9994fba8adbc3eef848984ab45
Reviewed-by: MohammadHossein Qanbari <mohammad.qanbari@qt.io>
(cherry picked from commit 8c94259cf15bae652662e9af90f5d9a93689bc60)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/labs/platform/qquicklabsplatformcolordialog.cpp    | 1 +
 src/labs/platform/qquicklabsplatformcolordialog_p.h    | 1 +
 src/labs/platform/qquicklabsplatformdialog.cpp         | 1 +
 src/labs/platform/qquicklabsplatformdialog_p.h         | 1 +
 src/labs/platform/qquicklabsplatformfiledialog.cpp     | 1 +
 src/labs/platform/qquicklabsplatformfiledialog_p.h     | 1 +
 src/labs/platform/qquicklabsplatformfolderdialog.cpp   | 1 +
 src/labs/platform/qquicklabsplatformfolderdialog_p.h   | 1 +
 src/labs/platform/qquicklabsplatformfontdialog.cpp     | 1 +
 src/labs/platform/qquicklabsplatformfontdialog_p.h     | 1 +
 src/labs/platform/qquicklabsplatformicon.cpp           | 1 +
 src/labs/platform/qquicklabsplatformicon_p.h           | 1 +
 src/labs/platform/qquicklabsplatformiconloader.cpp     | 1 +
 src/labs/platform/qquicklabsplatformiconloader_p.h     | 1 +
 src/labs/platform/qquicklabsplatformmenu.cpp           | 1 +
 src/labs/platform/qquicklabsplatformmenu_p.h           | 1 +
 src/labs/platform/qquicklabsplatformmenubar.cpp        | 1 +
 src/labs/platform/qquicklabsplatformmenubar_p.h        | 1 +
 src/labs/platform/qquicklabsplatformmenuitem.cpp       | 1 +
 src/labs/platform/qquicklabsplatformmenuitem_p.h       | 1 +
 src/labs/platform/qquicklabsplatformmenuitemgroup.cpp  | 1 +
 src/labs/platform/qquicklabsplatformmenuitemgroup_p.h  | 1 +
 src/labs/platform/qquicklabsplatformmenuseparator.cpp  | 1 +
 src/labs/platform/qquicklabsplatformmenuseparator_p.h  | 1 +
 src/labs/platform/qquicklabsplatformmessagedialog.cpp  | 1 +
 src/labs/platform/qquicklabsplatformmessagedialog_p.h  | 1 +
 src/labs/platform/qquicklabsplatformstandardpaths.cpp  | 1 +
 src/labs/platform/qquicklabsplatformstandardpaths_p.h  | 1 +
 src/labs/platform/qquicklabsplatformsystemtrayicon.cpp | 1 +
 src/labs/platform/qquicklabsplatformsystemtrayicon_p.h | 1 +
 30 files changed, 30 insertions(+)

diff --git a/src/labs/platform/qquicklabsplatformcolordialog.cpp b/src/labs/platform/qquicklabsplatformcolordialog.cpp
index 5f02c125fb..5dba4aafdd 100644
--- a/src/labs/platform/qquicklabsplatformcolordialog.cpp
+++ b/src/labs/platform/qquicklabsplatformcolordialog.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformcolordialog_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformcolordialog_p.h b/src/labs/platform/qquicklabsplatformcolordialog_p.h
index 59b2125e0d..408b9bdf43 100644
--- a/src/labs/platform/qquicklabsplatformcolordialog_p.h
+++ b/src/labs/platform/qquicklabsplatformcolordialog_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMCOLORDIALOG_P_H
 #define QQUICKLABSPLATFORMCOLORDIALOG_P_H
diff --git a/src/labs/platform/qquicklabsplatformdialog.cpp b/src/labs/platform/qquicklabsplatformdialog.cpp
index 8a7a60c738..a22b5080d4 100644
--- a/src/labs/platform/qquicklabsplatformdialog.cpp
+++ b/src/labs/platform/qquicklabsplatformdialog.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformdialog_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformdialog_p.h b/src/labs/platform/qquicklabsplatformdialog_p.h
index ca7acc47ac..7244bdd206 100644
--- a/src/labs/platform/qquicklabsplatformdialog_p.h
+++ b/src/labs/platform/qquicklabsplatformdialog_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMDIALOG_P_H
 #define QQUICKLABSPLATFORMDIALOG_P_H
diff --git a/src/labs/platform/qquicklabsplatformfiledialog.cpp b/src/labs/platform/qquicklabsplatformfiledialog.cpp
index 1eed73ade7..f4160819cc 100644
--- a/src/labs/platform/qquicklabsplatformfiledialog.cpp
+++ b/src/labs/platform/qquicklabsplatformfiledialog.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformfiledialog_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformfiledialog_p.h b/src/labs/platform/qquicklabsplatformfiledialog_p.h
index 879e6ed0b2..4d7519e21f 100644
--- a/src/labs/platform/qquicklabsplatformfiledialog_p.h
+++ b/src/labs/platform/qquicklabsplatformfiledialog_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMFILEDIALOG_P_H
 #define QQUICKLABSPLATFORMFILEDIALOG_P_H
diff --git a/src/labs/platform/qquicklabsplatformfolderdialog.cpp b/src/labs/platform/qquicklabsplatformfolderdialog.cpp
index d7a5435b48..56a0f5faaf 100644
--- a/src/labs/platform/qquicklabsplatformfolderdialog.cpp
+++ b/src/labs/platform/qquicklabsplatformfolderdialog.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformfolderdialog_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformfolderdialog_p.h b/src/labs/platform/qquicklabsplatformfolderdialog_p.h
index 19e40ab6f4..8ee3f17ec2 100644
--- a/src/labs/platform/qquicklabsplatformfolderdialog_p.h
+++ b/src/labs/platform/qquicklabsplatformfolderdialog_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMFOLDERDIALOG_P_H
 #define QQUICKLABSPLATFORMFOLDERDIALOG_P_H
diff --git a/src/labs/platform/qquicklabsplatformfontdialog.cpp b/src/labs/platform/qquicklabsplatformfontdialog.cpp
index f26884a687..052641f560 100644
--- a/src/labs/platform/qquicklabsplatformfontdialog.cpp
+++ b/src/labs/platform/qquicklabsplatformfontdialog.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformfontdialog_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformfontdialog_p.h b/src/labs/platform/qquicklabsplatformfontdialog_p.h
index 9913b496c4..c555b078fa 100644
--- a/src/labs/platform/qquicklabsplatformfontdialog_p.h
+++ b/src/labs/platform/qquicklabsplatformfontdialog_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMFONTDIALOG_P_H
 #define QQUICKLABSPLATFORMFONTDIALOG_P_H
diff --git a/src/labs/platform/qquicklabsplatformicon.cpp b/src/labs/platform/qquicklabsplatformicon.cpp
index 40e655616f..495a11c5a4 100644
--- a/src/labs/platform/qquicklabsplatformicon.cpp
+++ b/src/labs/platform/qquicklabsplatformicon.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2018 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformicon_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformicon_p.h b/src/labs/platform/qquicklabsplatformicon_p.h
index efd178a75b..43c40fc54a 100644
--- a/src/labs/platform/qquicklabsplatformicon_p.h
+++ b/src/labs/platform/qquicklabsplatformicon_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMICON_P_H
 #define QQUICKLABSPLATFORMICON_P_H
diff --git a/src/labs/platform/qquicklabsplatformiconloader.cpp b/src/labs/platform/qquicklabsplatformiconloader.cpp
index d96694c892..90d424eb43 100644
--- a/src/labs/platform/qquicklabsplatformiconloader.cpp
+++ b/src/labs/platform/qquicklabsplatformiconloader.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformiconloader_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformiconloader_p.h b/src/labs/platform/qquicklabsplatformiconloader_p.h
index 0fb406df4f..4b110fcc56 100644
--- a/src/labs/platform/qquicklabsplatformiconloader_p.h
+++ b/src/labs/platform/qquicklabsplatformiconloader_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMICONLOADER_P_H
 #define QQUICKLABSPLATFORMICONLOADER_P_H
diff --git a/src/labs/platform/qquicklabsplatformmenu.cpp b/src/labs/platform/qquicklabsplatformmenu.cpp
index 2f578ae76f..baa42f3bb9 100644
--- a/src/labs/platform/qquicklabsplatformmenu.cpp
+++ b/src/labs/platform/qquicklabsplatformmenu.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2020 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformmenu_p.h"
 #include "qquicklabsplatformmenubar_p.h"
diff --git a/src/labs/platform/qquicklabsplatformmenu_p.h b/src/labs/platform/qquicklabsplatformmenu_p.h
index d0286881fe..44d95bc0f1 100644
--- a/src/labs/platform/qquicklabsplatformmenu_p.h
+++ b/src/labs/platform/qquicklabsplatformmenu_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2020 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMMENU_P_H
 #define QQUICKLABSPLATFORMMENU_P_H
diff --git a/src/labs/platform/qquicklabsplatformmenubar.cpp b/src/labs/platform/qquicklabsplatformmenubar.cpp
index f1d4e8bd09..c5d4e38a93 100644
--- a/src/labs/platform/qquicklabsplatformmenubar.cpp
+++ b/src/labs/platform/qquicklabsplatformmenubar.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2020 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformmenubar_p.h"
 #include "qquicklabsplatformmenu_p.h"
diff --git a/src/labs/platform/qquicklabsplatformmenubar_p.h b/src/labs/platform/qquicklabsplatformmenubar_p.h
index 2f08f4eb27..06d323febb 100644
--- a/src/labs/platform/qquicklabsplatformmenubar_p.h
+++ b/src/labs/platform/qquicklabsplatformmenubar_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMMENUBAR_P_H
 #define QQUICKLABSPLATFORMMENUBAR_P_H
diff --git a/src/labs/platform/qquicklabsplatformmenuitem.cpp b/src/labs/platform/qquicklabsplatformmenuitem.cpp
index 8fb45694ab..78cd91fd93 100644
--- a/src/labs/platform/qquicklabsplatformmenuitem.cpp
+++ b/src/labs/platform/qquicklabsplatformmenuitem.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformmenuitem_p.h"
 #include "qquicklabsplatformmenu_p.h"
diff --git a/src/labs/platform/qquicklabsplatformmenuitem_p.h b/src/labs/platform/qquicklabsplatformmenuitem_p.h
index 92d80144e7..a093281883 100644
--- a/src/labs/platform/qquicklabsplatformmenuitem_p.h
+++ b/src/labs/platform/qquicklabsplatformmenuitem_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMMENUITEM_P_H
 #define QQUICKLABSPLATFORMMENUITEM_P_H
diff --git a/src/labs/platform/qquicklabsplatformmenuitemgroup.cpp b/src/labs/platform/qquicklabsplatformmenuitemgroup.cpp
index 822fd7db64..a3b3cd49c2 100644
--- a/src/labs/platform/qquicklabsplatformmenuitemgroup.cpp
+++ b/src/labs/platform/qquicklabsplatformmenuitemgroup.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2020 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformmenuitemgroup_p.h"
 #include "qquicklabsplatformmenuitem_p.h"
diff --git a/src/labs/platform/qquicklabsplatformmenuitemgroup_p.h b/src/labs/platform/qquicklabsplatformmenuitemgroup_p.h
index f4f1cc6ae3..6c65ea925a 100644
--- a/src/labs/platform/qquicklabsplatformmenuitemgroup_p.h
+++ b/src/labs/platform/qquicklabsplatformmenuitemgroup_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMMENUITEMGROUP_P_H
 #define QQUICKLABSPLATFORMMENUITEMGROUP_P_H
diff --git a/src/labs/platform/qquicklabsplatformmenuseparator.cpp b/src/labs/platform/qquicklabsplatformmenuseparator.cpp
index 58944bfe96..407ade3d5d 100644
--- a/src/labs/platform/qquicklabsplatformmenuseparator.cpp
+++ b/src/labs/platform/qquicklabsplatformmenuseparator.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformmenuseparator_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformmenuseparator_p.h b/src/labs/platform/qquicklabsplatformmenuseparator_p.h
index 364c592c20..8b08e9cc6a 100644
--- a/src/labs/platform/qquicklabsplatformmenuseparator_p.h
+++ b/src/labs/platform/qquicklabsplatformmenuseparator_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMMENUSEPARATOR_P_H
 #define QQUICKLABSPLATFORMMENUSEPARATOR_P_H
diff --git a/src/labs/platform/qquicklabsplatformmessagedialog.cpp b/src/labs/platform/qquicklabsplatformmessagedialog.cpp
index caefc13b79..5a2018cd13 100644
--- a/src/labs/platform/qquicklabsplatformmessagedialog.cpp
+++ b/src/labs/platform/qquicklabsplatformmessagedialog.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformmessagedialog_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformmessagedialog_p.h b/src/labs/platform/qquicklabsplatformmessagedialog_p.h
index f471601a65..d93772eb03 100644
--- a/src/labs/platform/qquicklabsplatformmessagedialog_p.h
+++ b/src/labs/platform/qquicklabsplatformmessagedialog_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMMESSAGEDIALOG_P_H
 #define QQUICKLABSPLATFORMMESSAGEDIALOG_P_H
diff --git a/src/labs/platform/qquicklabsplatformstandardpaths.cpp b/src/labs/platform/qquicklabsplatformstandardpaths.cpp
index 855c5f926f..7e7bd1e061 100644
--- a/src/labs/platform/qquicklabsplatformstandardpaths.cpp
+++ b/src/labs/platform/qquicklabsplatformstandardpaths.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformstandardpaths_p.h"
 
diff --git a/src/labs/platform/qquicklabsplatformstandardpaths_p.h b/src/labs/platform/qquicklabsplatformstandardpaths_p.h
index 80f4995129..266b83f9d1 100644
--- a/src/labs/platform/qquicklabsplatformstandardpaths_p.h
+++ b/src/labs/platform/qquicklabsplatformstandardpaths_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMSTANDARDPATHS_P_H
 #define QQUICKLABSPLATFORMSTANDARDPATHS_P_H
diff --git a/src/labs/platform/qquicklabsplatformsystemtrayicon.cpp b/src/labs/platform/qquicklabsplatformsystemtrayicon.cpp
index 825b7da6ad..4127d28efe 100644
--- a/src/labs/platform/qquicklabsplatformsystemtrayicon.cpp
+++ b/src/labs/platform/qquicklabsplatformsystemtrayicon.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qquicklabsplatformsystemtrayicon_p.h"
 #include "qquicklabsplatformmenu_p.h"
diff --git a/src/labs/platform/qquicklabsplatformsystemtrayicon_p.h b/src/labs/platform/qquicklabsplatformsystemtrayicon_p.h
index 3aa8c4be62..6aee14c8d2 100644
--- a/src/labs/platform/qquicklabsplatformsystemtrayicon_p.h
+++ b/src/labs/platform/qquicklabsplatformsystemtrayicon_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2017 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLABSPLATFORMSYSTEMTRAYICON_P_H
 #define QQUICKLABSPLATFORMSYSTEMTRAYICON_P_H
-- 
2.51.2

